source/install/install-kubernetes-mattermost.rst

```diff
@@ -40,47 +46,38 @@ The Mattermost installation manifest contains fields which must be edited in lin
-    "spec.elasticSearch", "The section for Elasticsearch settings. Remove this section if you will not be using Elasticsearch.", "No"
-    "spec.elasticSearch.host", "The hostname for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch", "No"
-    "spec.elasticSearch.username", "The username for your Elasticsearch instance.Remove this section if you will not be using Elasticsearch", "No"
-    "spec.elasticSearch.password", "The password for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch", "No"
+    "spec.licenseSecret", "The name of the Kubernetes secret containing your license (e.g. mattermost-license). Required for Enterprise deployments.", "No"
+    "spec.mattermostEnv", "List of custom environment variables for the Mattermost instance.", "No"
+    
+More fields are documented `in the example <https://github.com/mattermost/mattermost-operator/blob/master/docs/examples/mattermost_full.yaml>`__.
+If you have previous experience with Kubernetes Custom Resources you can also check the `Custom Resource Definition <https://github.com/mattermost/mattermost-operator/blob/master/config/crd/bases/installation.mattermost.com_mattermosts.yaml>`__.
 
-There are more advanced fields documented `here <https://raw.githubusercontent.com/mattermost/mattermost-operator/master/docs/examples/full.yaml>`__.
-
-Open a text editor and create a text file with the following details.
+Open a text editor and create a Mattermost installation manifest:
 
 .. code-block:: yaml
 
-  apiVersion: mattermost.com/v1alpha1
-  kind: ClusterInstallation
+  apiVersion: installation.mattermost.com/v1beta1
+  kind: Mattermost
   metadata:
-    name: mm-example-full
+    name: mm-example-full                         # Chose the desired name
   spec:
-    size: 5000users
-    ingressName: example.mattermost-example.com
+    size: 5000users                               # Adjust to your requirements
+    ingressName: example.mattermost-example.com   # Adjust to your domain
     ingressAnnotations:
       kubernetes.io/ingress.class: nginx
-    version: 5.14.0
-    mattermostLicenseSecret: ""
-    database:
-      storageSize: 50Gi
-    minio:
-      storageSize: 50Gi
-    elasticSearch:
-      host: ""
-      username: ""
-      password: ""
-
+    version: 5.28.0
+    licenseSecret: ""                             # If you have created secret in step 1, put its name here
+    
 Save the file as ``mattermost-installation.yaml``.
 
-To configure the Mattermost Installation with an external database and filestore refer to `Configuring external database and filestore (Recommended)`_ before proceeding.
-
+.. warning::
+  To configure the Mattermost Installation with an external database and filestore refer to `Configuring external database and filestore (Recommended)`_ before proceeding.
```

Comment:
I wonder if it would be better to just show how to configure external db and filestore in the step above. Then in this warning/note say you can use the mysql/minio operators but they're not recommended for production. What do you think?

Reply:
I merged the guide of configuring external db and filestore with the main one and added note, that those can be skipped.
Let me know what do you think.